Public/Src/Engine/UnitTests/Scheduler.IntegrationTest/BaselineTests.cs

```diff
@@ -1097,6 +1099,32 @@ public void MoveDirectoryFailed()
             SetExpectedFailures(2, 0);
         }
 
+        /// <summary>
+        /// Test to validate that global passthrough environment variables are visible to processes
+        /// </summary>
+        [Fact]
+        public void GlobalPassthroughEnvironmentVariables()
+        {
+            string testEnvironmentVariable = "ENV" + Guid.NewGuid().ToString().Replace("-", string.Empty);

```

Comment:
Should we add one more step in this case in the beginning, which testEnvironmentVariable is not set at all?

Reply:
Good suggestion. To save on test execution time I'll just set and query for another environment variable that I'm not passing in the global untracked scopes to make sure it isn't visible.